lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -124,7 +124,7 @@ def test_has_forum_access(self):
 
 @ddt.ddt
 @attr(shard=1)
-@mock.patch('student.models.UserProfile.has_registered', mock.Mock(return_value=True))
+@patch('openedx.stanford.djangoapps.sneakpeek.utils.is_registered', Mock(return_value=True))
```

Comment:
are the two imports from upstream, or is one a fork?

```python
import mock

from mock import Mock, patch
```

Reply:
We aren't forking any imports; we're just using what's available.